moz_security/sandboxes/heka/output/moz_security_ip_reputation.lua

```diff
@@ -0,0 +1,67 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+--[[
+# Mozilla Security IP Reputation Output
+
+Sends tsv of IP and violation weights to the IP Reputation Service (tigerblood) with HTTP with Hawk auth.
+
+## Sample Configuration
+
+```lua
+filename = "moz_telemetry_crash_summary.lua"
+message_matcher = "Type == 'inject_payload'"
```

Comment:
Heavy hitters will not be the only thing injecting tsv payloads

Reply:
What's an efficient way to only match the HH output?  Adding a filter on Logger like `&& Logger == 'analysis.hh_foo'`?

Reply:
Yeah you can match it by Logger (if you keep the analysis plugin).  Otherwise you will just consume the source data i.e. all the nginx log data `message_matcher = 'Type =~ \'nginx.access\'%'` and do everything in the output plugin.